src/applications/vaos/new-appointment/components/ReviewPage/VAAppointmentSection.jsx

```diff
@@ -23,12 +22,10 @@ export default function VAAppointmentSection({ data, facility }) {
             {TYPE_OF_VISIT.find(visit => visit.id === data.visitType)?.name}
           </div>
           <div>
-            <Link
-              to={newAppointmentFlow.visitType.url}
+            <va-link
+              href={newAppointmentFlow.visitType.url}
```

Comment:
use the `onClick` event instead of `href`

Reply:
@jenniemc Took care of this.